integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -73,6 +79,12 @@ export class OpfResourceLoaderService {
       };
 
       if (resource?.sri) {
+        const corsKeyvalue: OpfKeyValueMap | undefined =
+          resource?.attributes?.find(
+            (attr) => attr.key === 'crossorigin' && !!attr.value?.length
+          );
+        attributes['crossOrigin'] = corsKeyvalue?.value ?? 'anonymous';
```

Comment:
note that 'crossOrigin' (with capital O) is a property of Script entity (script.crossOrigin). not a typo prob.